test/model/store_test.dart

```diff
@@ -1070,10 +1156,13 @@ class LoadingTestGlobalStore extends TestGlobalStore {
   Map<int, List<Completer<PerAccountStore>>> completers = {};
 
   @override
-  Future<PerAccountStore> doLoadPerAccount(int accountId) {
+  Future<PerAccountStore> doLoadPerAccount(int accountId) async {
     final completer = Completer<PerAccountStore>();
     (completers[accountId] ??= []).add(completer);
-    return completer.future;
+    final store = await completer.future;
+    updateMachines[accountId] = UpdateMachine.fromInitialSnapshot(
+      store: store, initialSnapshot: eg.initialSnapshot());
+    return store;
```

Comment:
This is a bit unfortunate in that it complicates the picture for the existing tests that use this class. I think it's unnecessary if the new tests above just let go completely of the outer group's `preparePoll` and friends (as they've already half-done), as discussed in my preceding comment.